_data/drawings.yaml

```diff
@@ -1,3 +1,42 @@
-- title: Number ONE
-  description: Numero uno, little Juan
-  img_file: num1.png
+- title: Dad (2021)
+  description: 29.7 x 42cm, Pencil On Paper
+  img_file: Dad 2021.jpeg
+-title: Evening (2021)
+  description: 58 x 29cm, Pencil On Paper
+  img_file: Evening 2021.jpeg
+-title: Wellington Square (2021)
+  description: 48 x 41cm, Pencil On Paper
+  img_file: Wellington Square 2021.jpeg
+-title: Mundesley (2018)
+  description: 29.7 x 42xm, Pencil On Paper
+  img_file: Mundesley 2018.jpeg
+-title: Self Portrait (2021)
+  description: 17.5 x 24.5cm, Coloured Pencil On Paper 
+  img_file: Self Portrait 2021. jpeg
+-title: Untitled (2021)
+  description: 29.7 x 42cm 
+  img_file: Untitled 2021.jpeg
+-title: Little Clarendon Street (2021) 
+  description: 29.7 x 42cm, Coloured Pencil On Paper
+  img_file: Little Clarendon St. 2021.jpeg
+-title: Toys no.1 (2021)
+  description: 21 x 29.7, Pencil On Paper
+  img_file: Toys no.1 2021.jpeg
+-title: Toys no. 2 (2021)
+  description: 21 x 29.7cm, Pencil On Paper 
+  img_file: Toys no.2 2021.jpeg
+-title: Work (2021)
+  description: 88.5 x 48cm, Pen and Pencil On Paper 
+  img_file: Work 2021.jpeg
+-title: Still Life (2021)
+  description: 59 x 43cm, Spray Paint and Oil Pastel On Paper 
+  img_file: Still Life 2021.jpeg
+-title: Paulina (2021)
+  description: 53 x 42cm, Acrylic Paint and Oil Pastel On Paper 
+  img_file: Paulina 2021.jpeg
+-title: Pickle Lane (2019)
+  description: 21 x 29.7cm, Pen On Paper 
+  img_file: Pickle Lane 2019.jpeg
+-title: Amber Tides (2019)
+  description: 21 x 29.7cm, Pen On Paper
+  img_file: Amber Tides 2019.jpeg
```

Comment:
Are you aware the following added image files aren't listed?
- Lewis 2021
- Work in Colour 2021